### DISCUSSION TITLE
TSML UI error

### DISCUSSION THREAD

**Discussion by DanGMaher** [code4recovery/12-step-meeting-list#936] [General]
I got this error when opening aaspokane.org/meetings/ when using the TSML UI.  "Configuration error: time zone is not set."  When I switched back to Legacy UI, it went away.

**Comment by joshreisner** [comment]
Hi @DanGMaher yes you need to set the WordPress timezone in the general settings. In your case it should be set to `Los Angeles`

We will add a warning to the WP dashboard for sites that don't have this set.

**Reply by DanGMaher** [reply]
thanks Josh,  changed it to Los Angeles, it was set to manual UTC.  fixed the issue.
